src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.NoX86Intrinsics.xml

```diff
@@ -21,6 +24,42 @@
     <type fullname="System.Runtime.Intrinsics.X86.Avx2/X64">
       <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
     </type>
+    <type fullname="System.Runtime.Intrinsics.X86.Avx512BW">
```

Comment:
Unsure actually. Maybe some legacy thing from back when they weren't guaranteed false?

Reply:
It looks like I originally did it in https://github.com/dotnet/runtime/commit/ee12b54533c0bf7ce779677124715e1c8c769259, but looking back I'm not sure it is needed. Since we are just following existing convention here, let's not block this PR on it.

Reply:
https://github.com/dotnet/runtime/issues/83164